scarb/src/ops/resolve.rs

```diff
@@ -592,12 +603,17 @@ fn check_cairo_version_compatibility(packages: &[Package], ws: &Workspace<'_>) -
                     pkg.id.name,
                     package_version,
                     current_version
-                ));
+                );
+                if ignore_mismatch {
+                    ws.config().ui().warn(msg);
+                } else {
+                    ws.config().ui().error(msg);
```

Comment:
Should we add a help message suggesting to use newly added `--ignore-cairo-version` flag? 🤔

Reply:
As for the error messages, they do in fact suggest setting a flag. However, no warnings are displayed on mismatch 🤔

Reply:
So we can suggest setting it too :)

Reply:
Yea, I added that 😄 

My question was rather about the warnings: should we keep them or do as Cargo does? :D

Reply:
Keep please